Automattic/jetpack
# Update/boost plugin to use speed score package

Fixes 1201210512993648-as-1204651717398080

## Proposed changes:

* This PR udpates the Boost plugin to use the Boost Speed Score package added in #30914. I created this branched off of `add/jetpack-boost-speed-score-package` and I'm currently comparing this branch against trunk, but I'll rebase it as soon as #30914 is deployed.

### Other information:


## Jetpack product discussion


p1HpG7-k9o-p2

## Does this pull request change what data or activity we track or use?


Nope

## Testing instructions:






* Checkout this branch locally
* Run `jetpack build packages/boost-speed-score` and `jetpack install packages/boost-speed-score`
* Run `jetpack build plugins/boost` and `jetpack install plugins/boost`
* Alternatively, create a JN (or an Atomic Ephemeral) site with Jetpack Beta installed and with `update/boost-plugin-to-use-speed-score-package` as the selected branch
* Visit `/wp-admin/admin.php?page=jetpack-boost` and make sure Boost works as expected
